Dockerfile

```diff
@@ -0,0 +1,31 @@
+# ----------------------------------------------------------------------
+# First stage, compile application
+# ----------------------------------------------------------------------
+
+FROM node:14 AS builder
+
+WORKDIR /usr/src/app
+
+# Specify Mapbox Token to use
+ARG MAPBOX_TOKEN=""
+ENV MAPBOX_TOKEN=${MAPBOX_TOKEN}
+
+# copy only package for caching purposes
+COPY package*.json /usr/src/app/
+RUN npm install
+
+# copy rest of application
+COPY .eslintrc .prettierrc .lintstagedrc .huskyrc typedoc.json tsconfig.json babel.config.json *.js /usr/src/app/
+COPY src /usr/src/app/src/
+
+# build application
+RUN npm run build
+
+# ----------------------------------------------------------------------
+# Second stage, final image
+# ----------------------------------------------------------------------
+
+FROM nginx:alpine
```

Comment:
We might revisit this later and decide whether we want to serve the assets directly from a docker image. Depending on the server config, it might be better to use the docker file to just build the assets and serve them directly, instead of adding an extra layer between the main reverse proxy and the bundle.

For now, it's fine to keep this.